website/docs/r/iam_trusted_profiles_claim_rule.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profiles_claim_rule"
+description: |-
+  Manages iam_trusted_profiles_claim_rule.
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profiles_claim_rule
+
+Provides a resource for iam_trusted_profiles_claim_rule. This allows iam_trusted_profiles_claim_rule to be created, updated and deleted.
+
+## Example Usage
+
+```hcl
+resource "ibm_iam_trusted_profiles_claim_rule" "iam_trusted_profiles_claim_rule" {
+  conditions = { "claim" : "claim", "operator" : "operator", "value" : "value" }
+  profile_id = "profile_id"
+  type = "type"
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your resource.
+
+* `conditions` - (Required, List) Conditions of this claim rule.
+Nested scheme for **conditions**:
+	* `claim` - (Required, String) The claim to evaluate against.
+	* `operator` - (Required, String) The operation to perform on the claim. valid values are EQUALS, NOT_EQUALS, EQUALS_IGNORE_CASE, NOT_EQUALS_IGNORE_CASE, CONTAINS, IN.
```

Comment:
valid values are EQUALS, NOT_EQUALS, EQUALS_IGNORE_CASE, NOT_EQUALS_IGNORE_CASE, CONTAINS, IN.
as
Supported values are **EQUALS**, **NOT_EQUALS**, **EQUALS_IGNORE_CASE**, **NOT_EQUALS_IGNORE_CASE**, **CONTAINS**, **IN.**